spec/models/spree/order_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+require 'spree/order_decorator.rb'
+
+describe Spree::Order do
+  let(:order) { OrderWalkthrough.up_to(:payment) }
+  let(:stripe_apple_pay_payment_method) do
+    Spree::Gateway::StripeApplePayGateway.create!(
+      name: 'ApplePay',
+      preferred_domain_verification_certificate: 'dvc_test_jJG9kgDS75aADOsdfD1fDxQ'
+    )
+  end
+  let(:stripe_credit_card_payment_method) do
+    Spree::Gateway::StripeGateway.create!(
+      name: 'CreditCard',
+      preferred_secret_key: 'sk_test_aFQiNCcaOlXgDIwJHDdWlGN',
+      preferred_publishable_key: 'pk_test_ZItEjGEZxRiCuzGZHdsbOaj'
```

Comment:
I don't think it's a good idea to hard-code publishable keys in tests 🙄

Reply:
It's required to create a `Spree::Gateway::StripeGateway `, and these keys are just random string.

Reply:
Can't you use `FFAker` here for some random string generation?

Reply:
I can, and I will if you insist, but the reason I did it, is because we do the same in other similar cases, i.e.: https://github.com/spree/spree_gateway/blob/master/spec/models/gateway/usa_epay_transaction_spec.rb#L7

Reply:
@dimon-k let's use ffaker data - that old spec should be revisited in the future - we shouldn't commit any credentials ;)

Reply:
fixed